### PR TITLE
Add newsletter Feb

### PR DESCRIPTION
**Pull request by wipfli**
This is a newsletter about things which have been going on at MapLibre in the last month.

Feel free to add your points to the existing sections or create a new section for other projects in the MapLibre github organization.

### CODE REVIEW

File: content/news/2023-02-14-maplibre-newsletter-february-2023.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- [birkskyum](https://github.com/maplibre/maplibre-gl-native/commits?author=birkskyum) updated AGP.
+- [alanchenboy](https://github.com/maplibre/maplibre-gl-native/commits?author=alanchenboy) contributed 2 pull requests to fix a snapshotting crash and enable C++ unit tests.
+
+## MapLibre GL JS
+
+In the last Technical Steering Committee meeting on Feb 8th, 2023, we had the pleasure to welcome some new faces, including people from OSM Americana and Felt.
+
+There were some discussions around how to move the style specification forward and how the contribution process can be more stream-lined.
+
+We publish MapLibre GL JS v3.0.0-pre.4 on NPM. Try it out and let us know what you think. All bug reports are welcome! The new features, bug fixes and breaking changes are documented in the [CHANGELOG](https://github.com/maplibre/maplibre-gl-js/blob/main/CHANGELOG.md).
```
Comment (louwers) [inline content/news/2023-02-14-maplibre-newsletter-february-2023.md:49]:
```suggestion
We published MapLibre GL JS v3.0.0-pre.4 on npm. Try it out and let us know what you think. All bug reports are welcome! The new features, bug fixes and breaking changes are documented in the [CHANGELOG](https://github.com/maplibre/maplibre-gl-js/blob/main/CHANGELOG.md).
```

File: content/news/2023-02-14-maplibre-newsletter-february-2023.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+### Bounties
+
+MapLibre GL Native published Bounties for code development tasks. Read our step-by-step guide [LINK] to learn how you can work on Bounties and get paid.
+
+Currently, the following Bounties are available (see [here](https://github.com/maplibre/maplibre-gl-native/issues?q=is%3Aissue+is%3Aopen+label%3A%22%F0%9F%92%B0+bounty+L%22%2C%22%F0%9F%92%B0+bounty+S%22%2C%22%F0%9F%92%B0+bounty+M%22+) for an up-to-date list):
+
+- Documentation: Update the Android test app: [#741](https://github.com/maplibre/maplibre-gl-native/issues/741)
+
+### GitHub
```
[the remark this answers is not in the document]
Comment (wipfli) [reply content/news/2023-02-14-maplibre-newsletter-february-2023.md:31]: Sure, can you "suggest changes"

File: content/news/2023-02-14-maplibre-newsletter-february-2023.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Code contributions since January 15th, 2023:
+
+- [louwers](https://github.com/maplibre/maplibre-gl-native/commits?author=louwers) contributed 20 pull requests focusing on improved continuous integration and documentation.
```
[the remark this answers is not in the document]
Comment (wipfli) [reply content/news/2023-02-14-maplibre-newsletter-february-2023.md:35]: I like the number

[the remark this answers is not in the document]
Comment (wipfli) [reply content/news/2023-02-14-maplibre-newsletter-february-2023.md:35]: But @louwers, maybe you can better summarize the content of the pull requests?

File: content/news/2023-02-14-maplibre-newsletter-february-2023.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+- [tdcosta100](https://github.com/maplibre/maplibre-gl-native/commits?author=tdcosta100) contributed 4 pull requests to prepare Windows support.
+- [KiwiKilian](https://github.com/maplibre/maplibre-gl-native/commits?author=KiwiKilian) improved the node documentation.
+- [fynngodau](https://github.com/maplibre/maplibre-gl-native/commits?author=fynngodau) contributed 2 pull requests to improve Kotlin code and fix an issue with bitmaps.
+- [hallahan](https://github.com/maplibre/maplibre-gl-native/commits?author=hallahan) improved the iOS docs.
+- [birkskyum](https://github.com/maplibre/maplibre-gl-native/commits?author=birkskyum) updated AGP.
```
[the remark this answers is not in the document]
Comment (wipfli) [reply content/news/2023-02-14-maplibre-newsletter-february-2023.md:40]: Maybe android gradle plugin?

File: content/news/2023-02-14-maplibre-newsletter-february-2023.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+## MapLibre GL Native
+
+We are happy to welcome Bart Louwers and Ovidiu Voda as Maintainers of MapLibre GL Native. They started in January 2023 and we can see how their work has a positive impact on MapLibre GL Native with many new people contributing to the repo. Thank you both for your efforts!
+
+MapLibre GL Native for Android [v10.0.0-pre.0](https://github.com/maplibre/maplibre-gl-native/releases/tag/android-v10.0.0-pre.0) is out now. Try it out and let us know what you think. All bug reports are welcome! The new features, bug fixes and breaking changes are documented in the [release notes](https://github.com/maplibre/maplibre-gl-native/releases/tag/android-v10.0.0-pre.0).
+
+### Bounties
+
+MapLibre GL Native published Bounties for code development tasks. Read our step-by-step guide [LINK] to learn how you can work on Bounties and get paid.
```
[the remark this answers is not in the document]
Comment (wipfli) [reply content/news/2023-02-14-maplibre-newsletter-february-2023.md:29]: Thanks, need to add that

File: content/news/2023-02-14-maplibre-newsletter-february-2023.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+In the last Technical Steering Committee meeting on Feb 8th, 2023, we had the pleasure to welcome some new faces, including people from OSM Americana and Felt.
+
+There were some discussions around how to move the style specification forward and how the contribution process can be more stream-lined.
+
+We publish MapLibre GL JS v3.0.0-pre.4 on NPM. Try it out and let us know what you think. All bug reports are welcome! The new features, bug fixes and breaking changes are documented in the [CHANGELOG](https://github.com/maplibre/maplibre-gl-js/blob/main/CHANGELOG.md).
+
+### Bounties
+
+MapLibre GL JS published Bounties for code development tasks. Read our step-by-step guide [LINK] to learn how you can work on Bounties and get paid.
```
[the remark this answers is not in the document]
Comment (wipfli) [reply content/news/2023-02-14-maplibre-newsletter-february-2023.md:57]: Thanks, need to add that

File: content/news/2023-02-14-maplibre-newsletter-february-2023.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (louwers) [reply content/news/2023-02-14-maplibre-newsletter-february-2023.md:39]: Sure.

[the remark this answers is not in the document]
Comment (louwers) [reply content/news/2023-02-14-maplibre-newsletter-february-2023.md:39]:
```
- [louwers](https://github.com/maplibre/maplibre-gl-native/commits?author=louwers) contributed 20 pull requests.
  - Continuous integration for Android is now green again. We no longer use a custom Docker image.
  - The various README files throughout the repository where refreshed.
  - The [Android API docs](https://maplibre.org/maplibre-gl-native/android/api/) now use Dokka, which is a documentation generator with better support for Kotlin than the previously used Javadoc.
  - New [Core C++ docs](https://maplibre.org/maplibre-gl-native/cpp/api/) were published as well, using Doxygen as documentation generator.
  - C++17 is used for compilation and some parts of the codebase were updated to take advantage of this. 
- [tdcosta100](https://github.com/maplibre/maplibre-gl-native/commits?author=tdcosta100) contributed 4 pull requests to prepare Windows support.
- [KiwiKilian](https://github.com/maplibre/maplibre-gl-native/commits?author=KiwiKilian) improved the node documentation.
- [fynngodau](https://github.com/maplibre/maplibre-gl-native/commits?author=fynngodau) contributed 2 pull requests to improve Kotlin code and fix an issue with bitmaps.
- [hallahan](https://github.com/maplibre/maplibre-gl-native/commits?author=hallahan) improved the iOS docs.
- [birkskyum](https://github.com/maplibre/maplibre-gl-native/commits?author=birkskyum) updated Android Gradle Plugin.
- [alanchenboy](https://github.com/maplibre/maplibre-gl-native/commits?author=alanchenboy) contributed 2 pull requests to fix a snapshotting crash and enable C++ unit tests on macOS. 
```

I added some more details.

File: content/news/2023-02-14-maplibre-newsletter-february-2023.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+Currently, the following Bounties are available (see [here](https://github.com/maplibre/maplibre-gl-native/issues?q=is%3Aissue+is%3Aopen+label%3A%22%F0%9F%92%B0+bounty+L%22%2C%22%F0%9F%92%B0+bounty+S%22%2C%22%F0%9F%92%B0+bounty+M%22+) for an up-to-date list):
+
+- Documentation: Update the Android test app: [#741](https://github.com/maplibre/maplibre-gl-native/issues/741)
+
+### GitHub
+
+Code contributions since January 15th, 2023:
+
+- [louwers](https://github.com/maplibre/maplibre-gl-native/commits?author=louwers) contributed 20 pull requests focusing on improved continuous integration and documentation.
```
Comment (wipfli) [inline content/news/2023-02-14-maplibre-newsletter-february-2023.md:39]:
```suggestion
- [louwers](https://github.com/maplibre/maplibre-gl-native/commits?author=louwers) contributed 20 pull requests.
  - Continuous integration for Android is now green again. We no longer use a custom Docker image.
  - The various README files throughout the repository where refreshed.
  - The [Android API docs](https://maplibre.org/maplibre-gl-native/android/api/) now use Dokka, which is a documentation generator with better support for Kotlin than the previously used Javadoc.
  - New [Core C++ docs](https://maplibre.org/maplibre-gl-native/cpp/api/) were published as well, using Doxygen as documentation generator.
  - C++17 is used for compilation and some parts of the codebase were updated to take advantage of this. 

```

**Comment by wipfli** [comment]
Ready for review

**Review by louwers — APPROVED** [review]

**Comment by louwers** [comment]
One more thing: maybe the upcoming events and events can be merged.

But up to you.

**Comment by wipfli** [comment]
https://twitter.com/maplibre/status/1626641184159563776

**Comment by wipfli** [comment]
https://www.linkedin.com/feed/update/urn:li:activity:7032407610444181504

**Pull request merged**